src/utils/Processing.ts

```diff
@@ -40,6 +42,14 @@ export async function getResources(
   return resources;
 }
 
+export async function getConfig(
+  processor: FHIRProcessor,
+  externalDeps: string[]
+): Promise<ExportableConfiguration> {
+  const config = processor.processConfig(externalDeps);
+  return config;
+}
+
```

Comment:
I don't think this function is `async` anymore, but honestly, this function is so simple now that I don't think it's necessary anymore.  It's just as easy to invoke `processor.processConfig(externalDeps)` as it is to invoke `getConfig(processor, externalDeps)`.  So I'd just remove this extra layer of indirection.